### ISSUE TITLE
libModSecurity(ver.3) SecAuditLogParts bug found

### ISSUE THREAD

**Issue by bosungs2y** [SpiderLabs/ModSecurity#1499]
There is a bug in SecAuditLogParts C, F option.
Setting modsecurity.conf to the following setting does not work as expected

# Information
Nginx(1.11.13)
libModSecurity(ver.3)

# Modsecurity.conf
> SecRequestBodyAccess On
> SecAuditLogParts ABCJFEZ

# Curl Request
` curl -X POST  http://10.10.10.10/index.html --data 'kkkkkkkkkkkk'`

# Result (libsecurity - log )
```
---uUjFJBdx---A--
[13/Jul/2017:17:46:43 +0900] 149993560396.395206 172.26.137.77 54113 10.10.10.10 80
---uUjFJBdx---B--
GET /index.html HTTP/1.1
Host: http://10.10.10.10
User-Agent: curl/7.51.0
Accept: */*
Content-Length: 51
Content-Type: application/x-www-form-urlencoded

---uUjFJBdx---C--

---uUjFJBdx---E--
<!DOCTYPE html>

<html><body><h1>It works!</h1></body></html>
<h2><span style="color:blue"; >ModSecurity Test Page (Nginx.ver)<span></h2>

---uUjFJBdx---F--
Server: nginx/1.11.13
Date: Thu, 13 Jul 2017 08:46:43 GMT
Content-Length: 379
Content-Type: text/html
Last-Modified: Wed, 12 Jul 2017 06:43:33 GMT
Connection: keep-alive
ETag: "5965c515-17b"
Accept-Ranges: bytes

---uUjFJBdx---J--

---uUjFJBdx---Z--
```

# Bug
C option does not include the request body (ex : kkkkkkkkkkkk)
F option does not include response_code(ex : HTTP/1.1 200 OK)

**Comment by zimmerle** [comment]
The support for section "C" was implemented at: 43cb8ed652aeb145dd968b7a6c5e853e24fc8f89

**Comment by zimmerle** [comment]
The "F" section was fixed at: 1f1e8324b1838ad51111654948885d0db7a5fc96
